wordpress-mobile/WordPress-iOS
# Fix existing UI Login Tests

This PR updates the existing Login Tests in WordPressUITests so they pass. We currently have two tests:
* Simple login/logout (with email/password)
* Unsuccessful login

The updates rely on changes to the accessibility identifiers in the WordPressAuthenticator pod. The same identifiers are used to generate screenshots, so I made some updates there as well.

To test:

Add `~/.wpcom_test_credentials` with the following details:

```
testUserEmail=YOUR@EMAIL
testUserPassword=YOUR_PASSWORD
```

Then:
* Run the tests in `LoginTests.swift` and confirm they pass.
* Run the screenshot generation test in `WordPressScreenshotGeneration.swift` and confirm it passes.
